resources/lang/en.json

```diff
@@ -1,5 +1,6 @@
 {
     "cpu_usage": "CPU usage",
     "ram_usage": "RAM usage",
+    "disk_usage": "DISK usage",
```

Comment:
CPU and RAM are capitalized since they're acronyms. Since disk isn't an acronym, it should just be `Disk`. This applies for all translations.